getting-started-volunteer.md

```diff
@@ -0,0 +1,26 @@
+# I want to volunteer, where do I start?
+
+So you know what [we are about](https://github.com/djangonaut-space/pilot-program/blob/main/README.md), you have read our [code of conduct](https://github.com/djangonaut-space/pilot-program/blob/main/conduct.md), and you are interested in volunteering, so what now? First of all, thank you! Our community thrives because of people like you. 
+
+### Types of Volunteers
+There are a few ways to volunteer with this community: as a [Navigator](https://github.com/djangonaut-space/pilot-program/blob/main/navigators.md), [Captain](https://github.com/djangonaut-space/pilot-program/blob/main/captains.md), or Organizer. We have summarized those roles here as an overview but please read their detailed guidelines to get a more in-depth understanding of each. 
+
+**Navigator:**
+- Navigators can expect to commit 2-4 hours of their time each week over the 3 month session. 
+- This would involve showing up to our virtual meetings/co-working hours and assisting other members or helping out members in the discord channel. 
+- More details: [Navigator Guideline](https://github.com/djangonaut-space/pilot-program/blob/main/navigators.md)
+
+
+**Captain:**
+- Attend welcome and closing sessions and be available on discord.
+- Check-in with Navigators and Djangonauts regularly throughout the program. 
+- Commit 3-5 hours a week for 3 months. 
+- More details: [Captain Guidelines](https://github.com/djangonaut-space/pilot-program/blob/main/captains.md)
+
+**Organizer:**
+- Responsible for organizing and leading meetings, update statuses on our website and GitHub, organization communication, Code of Conduct follow through, recruiting volunteers.
```

Comment:
Any issues with changing it to "and recruiting volunteers."? I'm a fan of the oxford comma, but I don't need it.